### PR TITLE
Features/website banner

### PR DESCRIPTION
**Pull request by immangat**
his pull request introduces a new React component called "Banner" to the project. The component includes a description and a one-liner for OpenBC.

Image of the Banner:
image

### CODE REVIEW

File: src/components/Banner/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+interface BannerProps {
+
+}
+
+function Banner() {
+  return (
+    <Box
+      sx={{
+        textAlign: 'center',
```
Comment (garenpham) [inline src/components/Banner/Banner.tsx:13]: We can separate the styling into `.ts` files in /src/styles folder.

**Review by garenpham — APPROVED** [review]
Apart from moving the styles to the separate folder /src/styles, the rest are looking great to me. Well done!

**Comment by SamHuo213** [comment]
I noticed that this has changes from @garenpham 's PR. I guess you branched from his.

When the previous PR that you branch from has not been merged to main. I suggest you point to that branch, it makes it easier for reviewers.

image

**Comment by SamHuo213** [comment]
Thanks @immangat 

I see @garenpham approved. Looks good to me then.

**Pull request merged**